apt_shared.py

```diff
@@ -90,10 +90,17 @@ def checkData(testConfig):
             except IOError as e:
                 logMsg(testConfig['LOG_FILE'], "FAILED TO OPEN LOCAL REPORT FILE: " + sessionData['LOCAL_SESSION_FILE'])
                 continue
-            for item in target['SUCCESS_LIST']:
-                if item not in fileContents:
-                    logMsg(testConfig['LOG_FILE'], str(item))
-                    statusFlag = False
+            if 'SUCCESS_LIST' in target:
+                for item in target['SUCCESS_LIST']:
+                    if item not in fileContents:
+                        logMsg(testConfig['LOG_FILE'], str(item))
+                        statusFlag = False
+            if 'FAILURE_LIST' in target:
+                for item in target['FAILURE_LIST']:
+                    for line in fileContents.splitlines():

```

Comment:
Why does this process differently?

Should there be a requirement that at least one of `SUCCESS_LIST` or `FAILURE_LIST ` exists?

Reply:
That's a good point, and I thought about it, but that would be more of a non-trivial change that should exist in the `verifyConfig` method.  Currently, that method only deals in exclusive entries.